libs/remix-core-plugin/src/lib/compiler-content-imports.ts

```diff
@@ -117,7 +118,18 @@ export class CompilerImports extends Plugin {
     * @returns {Promise} - string content
     */
   async resolveAndSave (url, targetPath) {
-    if (url.indexOf('remix_tests.sol') !== -1) return remixTests.assertLibCode
+    if (url.indexOf('remix_tests.sol') !== -1 || url.indexOf('remix_accounts.sol') !== -1) {
+      let content
+      if (url === 'remix_tests.sol') content = assertLibCode
+      else if (url === 'remix_accounts.sol') {
+        this.testRunner = new UnitTestRunner()
```

Comment:
getting a static content from remix-tests in that case is fine, but instantiating `new UnitTestRunner()` and re-executing `init` is not optimal _at all_. 
Moreover `init` include the instantiation of a simulator `this.createWeb3Provider(web3)`, which won't be used in that case, but will instanciate a full js-vm, that looks like wasting resources.

instead of that, would this work to have `await this.call('activate', 'solidityUnitTesting)'`? please check the syntax.